BrainCloudClient/Assets/BrainCloud/Client/BrainCloud/Internal/RTTComms.cs

```diff
@@ -390,6 +385,7 @@ private void Websocket_OnOpen(BrainCloudWebSocket accepted)
             {
                 m_clientRef.Log("RTT: Connection established.");
             }
+            m_sinceLastHeartbeat = DateTime.Now.TimeOfDay;
```

Comment:
actually, move that to where this is set: 
```
m_rttConnectionStatus = RTTConnectionStatus.CONNECTED;
```
I believe it's in update, then it's looking through "connect" in a switch case.